crates/nargo_cli/tests/rebuild.sh

```diff
@@ -3,30 +3,42 @@ set -e
 
 excluded_dirs=("workspace" "workspace_default_member")
 
-cd ./execution_success
+current_dir=$(pwd)
+base_path="$current_dir/execution_success"
+
+# Ensure the base acir_artifacts directory exists
+mkdir -p $current_dir/acir_artifacts
 
 # Loop over every directory
-for dir in ./*; do
+for dir in $base_path/*; do
   if [[ ! -d $dir ]]; then
     continue
   fi
 
   dir_name=$(basename "$dir")
+
   if [[ ! " ${excluded_dirs[@]} " =~ " ${dir_name} " ]]; then
+      echo "Creating directory $current_dir/acir_artifacts/$dir_name"
+      mkdir -p $current_dir/acir_artifacts/$dir_name
+      echo "Directory created."
```

Comment:
```suggestion
      if [[ ! -d "$current_dir/acir_artifacts/$dir_name" ]]; then
        echo "Creating directory $current_dir/acir_artifacts/$dir_name"
        mkdir -p $current_dir/acir_artifacts/$dir_name
        echo "Directory created."
      fi
```

To avoid some console spam. (I haven't tested this snippet)

Reply:
Removed